.vscode/settings.json

```diff
@@ -1,9 +0,0 @@
-{
```

Comment:
I had originally removed this b/c I have a local extension that modifies this, but I see it's actually in `main` and we probably don't want to remove it.. I'll just deal with it myself for now and maybe ask the SDK folks if we really need it...

Reply:
added it back.